packages/account/src/Sections/Profile/PhoneVerification/verification-link-expired-modal.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Modal, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useHistory } from 'react-router';
+import { observer, useStore } from '@deriv/stores';
+import { LabelPairedCircleXmarkLgRegularIcon } from '@deriv/quill-icons';
+
+type TVerificationLinkExpiredModal = {
+    should_show_verification_link_expired_modal: boolean;
+    setShouldShowVerificationLinkExpiredModal: (value: boolean) => void;
+};
+
+const VerificationLinkExpiredModal = observer(
+    ({
+        should_show_verification_link_expired_modal,
+        setShouldShowVerificationLinkExpiredModal,
+    }: TVerificationLinkExpiredModal) => {
+        const history = useHistory();
+        const handleCancelButton = () => {
+            setShouldShowVerificationLinkExpiredModal(false);
+            history.goBack();
+        };
+        const { ui } = useStore();
+        const { is_mobile } = ui;
```

Comment:
Should we use `isMobile` from `useDevice` hook?

Reply:
Tried to implement the useDevice here but it's giving me error: 
![Screenshot 2024-05-17 at 3 20 28 PM](https://github.com/suisin-deriv/deriv-app/assets/103026762/da8847d7-aa56-439c-b55e-1974e1c018f4)
last time i had this error when implementing the quill-ui, maybe need to ask the deriv-com/ui team to help solve this first

Reply:
yes, let's ask them 🙏